modules/persistent-storage-csi-sc-multiple-none.adoc

```diff
@@ -0,0 +1,50 @@
+// Module included in the following assemblies:
+//
+// persistent-storage-csi-sc-manage
+//
+:_content-type: PROCEDURE
+[id="persistent-storage-csi-sc-multiple-none_{context}"]
+= Absent or multiple default storage classes
+
+== Multiple default storage classes
+Multiple default storage classes can occur if you mark a non-default storage class as default and do not unset the existing default storage class, or you create a default storage class when a default storage class is already present. With multiple default storage classes present, any persistent volume claim (PVC) requesting the default storage class (`pvc.spec.storageClassName`=nil) gets the most recently created default storage class, and the administrator receives an alert that there are multiple default storage classes.
```

Comment:
To avoid confusion, the effective storage class is the most recently created storage class, not the most recent SC that has been set as default. We're looking at the SC metadata.creationTimestamp.

Reply:
@gcharot I assume that this statement refers only to situations where there are multiple default storage classes. In such cases, when a PVC tries to request the default sc, it then gets the most recently created SC regardless of whether or not that SC has default status?